File: ios/RCTMGL-v10/MBXLightViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(MBXLight, MBXLightViewManager, RCTViewManager)

// light props
RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RCTMGL-v10/MBXLightViewManager.swift
@objc(MBXLightViewManager)
class MBXLightViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let light = MBXLight()
      light.bridge = self.bridge
      return light
    }
}
